core/image.php

```diff
@@ -27,7 +31,8 @@ public function __construct($id) {
         }       
     }
     
-    public function render($size="original", $class="") {
+    public function render($size="original", $class="", $format="original") {
+        // TODO: allow system-wide pref for default image encoding
```

Comment:
Original will serve up png/jpeg/webp as uploaded - but might be cool to force 'next-gen' across the board?

Reply:
Would be very cool. Could setting be added to config or is that needless complication?

Reply:
> Would be very cool. Could setting be added to config or is that needless complication?

Simplest would be in config.php - assume jpeg unless otherwise stated. Could also make it part of general settings if needed - doing it that way would allow us to change the 'default' with a release vs having to manually update config files everywhere.